.github/workflows/docker-tests.yml

```
@@ -0,0 +1,25 @@
+name: Platform PR CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "**" ]
```

If we do:

```suggestion
  push:
    branches: [ "**" ]
```

we'll start as soon as you push, vs. just when you create a PR.

Although thinking about it we do maintain a long term non-PR branch, `dev`, and I'm not sure if CI would pass for it. It doesn't matter if it doesn't but it'd show us a little red `x`.

Figured I'd mention it, but feel free to ignore!

---

Yeah I was trying to avoid any breakages on the `dev` branch since it won't have any of the latest changes. We may want to figure out a longer term fix for the `dev` branch since it's going to get further away from `main` over time. I don't think it's necessary at the moment.

I'll go with your suggestion and if it causes too much confusion or friction in the future we can just put it back. Definitely not a big deal!